package.json

```diff
@@ -67,9 +69,12 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/mauriciolauffer/wdio-qunit-service.git"
+    "url": "https://github.com/yottaa/wdio-qunit-service.git"
+  },
+  "publishConfig": {
+    "registry": "https://npm.pkg.github.com"
   },
   "bugs": {
-    "url": "https://github.com/mauriciolauffer/wdio-qunit-service/issues"
+    "url": "https://github.com/pmoeller91-yottaa/wdio-qunit-service/issues"
```

Comment:
should change to yottaa? or delete i guess

Reply:
Fixed!